pkgs/esp-idf/tools.nix

```diff
@@ -81,7 +87,15 @@ let
 
       phases = [ "unpackPhase" "installPhase" ];
 
-      installPhase = ''
+      installPhase = let
+        wrapCmd = if system == "linux-x86_64" then
+        ''
+          makeWrapper ${fhsEnv}/bin/${pname}-env $FILE_PATH --add-flags "$FILE_PATH-unwrapped" ${lib.strings.concatStringsSep " " exportVarsWrapperArgsList}
```

Comment:
I actually don't know if this whole thing is required, given what you've done for other platforms. For now I think it's fine, as long as it works for all the supported platforms.

Reply:
Yeah this was a bit of a hack, I didn't want to disturb what was already there in case it was load bearing :)